source/config/__init__.py

```diff
@@ -474,35 +474,6 @@ def setStartOnLogonScreen(enable: bool) -> None:
 			raise RuntimeError("Slave failed to set startOnLogonScreen")
 
 
-def addConfigDirsToPythonPackagePath(module, subdir=None):

```

Comment:
Can you please retain backwards compatibility here? https://github.com/nvaccess/nvda/blob/master/devDocs/deprecations.md